backend/de.metas.edi/src/main/java/de/metas/edi/spi/impl/EdiInvoiceCopyHandler.java

```diff
@@ -50,7 +52,7 @@ public void copyValues(@NonNull final org.compiere.model.I_C_Invoice from, @NonN
 		
 		// task 08926
 		// Make sure the EdiEnabled flag is inherited if the invoice is created form another invoice
-		toToUse.setIsEdiEnabled(fromToUse.isEdiEnabled());
+		ediDocumentBL.setEdiEnabled(toToUse, fromToUse.isEdiEnabled());
```

Comment:
also pls makes sure IsEDIEnabled and EDI Status columns are flagged as calculated/do not copy!